src/components/SkillsPanel.js

```diff
@@ -1,7 +1,8 @@
-import React, { useState } from 'react';
+/* eslint-disable */
```

Comment:
:angry: 

lol I guess this got left behind?

Reply:
Yup 😅 will tidy up